spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -19,7 +19,25 @@
         model = define_model :example do
           def self.statuses; end
         end
-        message = "Expected #{model} to define :statuses as an enum"
+        message = "Expected #{model} to define :statuses as an enum and store the value in a column with an integer type"
+        assertion = -> {
+          expect(model.new).to define_enum_for(:statuses)
+        }
+        expect(&assertion).to fail_with_message(message)
+      end
+    end
+
+    context "if the column storing the attribute is not an integer type" do
+      it "rejects" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.